Modules/SubModule_Population/xASL_wrp_CreatePopulationTemplates.m

```diff
@@ -294,9 +294,13 @@ function xASL_wrp_CreatePopulationTemplates(x, bSaveUnmasked, bCompute4Sets, Spe
 for iScanType=1:length(PreFixList)
     UnAvailable = 0;
 
-	% Define sessions
+	% Define sessions if relevant for the datatype
```

Comment:
This defaulting to ASL_1 is something we need in every instance right, so I would remove it here and move it to `xASL_adm_GetPopulationSessions` (where I perhaps already programmed it)

Reply:
Slightly different thing. In `xASL_adm_GetPopulationSessions` we are looking for `ASL_X` sessions. So things like `M0_ASL_2` or `CBF_ASL_3`. But it won't return any sessions for T1w images for example, as they are not bound to sessions.

So for T1w, `xASL_adm_GetPopulationSessions` will return an empty list of sessions and nSessions = 1. But here, we don't call this function, as we are not interested in session lists for dataTypes like T1w. So it's better that this stays here.

Reply:
So while I agree with how you edit `xASL_adm_GetPopulationSessions` and will keep it. I would not call this function here. Because it will generate an unnecessary warning for missing session for a case where we are not even expecting that sessions exist.